scripts/check_correct.py

```diff
@@ -0,0 +1,35 @@
+def correct(src, target):
```

Comment:
as this script is in a folder merely called "scripts" it should have a more descriptive name, check_correct for what?